File: osb-core-main/Platform/Integration/Osb.Core.Platform.Integration.Entity/FitBank/Models/Request/InactivateAndReissueCardRequest.cs
using Osb.Core.Platform.Integration.Entity.Models.Request.Base;

namespace Osb.Core.Platform.Integration.Entity.FitBank.Models.Request
{
    public class InactivateAndReissueCardRequest : BaseRequest
    {
        public new string Method { get => "InactivateAndReissueCard"; }
        public string IdentifierCard { get; set; }
        public string Pin { get; set; }
        public string ReasonCode { get; set; }
        public CardDeliveryAddressRequest CardDeliveryAddress { get; set; }
    }
}

File: osb-mobile-main/osb-mobile-main/src/features/card/redux/models/request/InactivateAndReissueCardRequest.ts
import { ApiRequest } from '_config/api'
import { ReasonCode } from 'features/card/redux/models/reasonCodeEnum'
import { CardDeliveryAddress } from 'features/card/redux/models/CardDeliveryAddress'

export interface InactivateAndReissueCardRequest extends ApiRequest {
  identifierCard: string
  pin: string
  reasonCode: ReasonCode
  cardDeliveryAddress: CardDeliveryAddress
}
